cli/src/cli.rs

```diff
@@ -68,11 +72,14 @@ pub enum Commands {
         #[clap(short, long, action)]
         last_finalization_proof: String,
     },
-    /// Print the information about the block data Git server that this node is hosting.
+    /// Print the information about the Git server that this node is hosting.
     Git,
     /// Clean the repository, removing all the outdated (incompatible with `finalized`) commits.
     Clean {
-        /// If enabled, it will remove all the commits except those included in `finalized`.
+        /// If enabled, it will remove
```

Comment:
I've seen the term "enabled" used for options of a CLI command, but not for the CLI command itself.
How about "If the command is run,"?

Reply:
This is description about the option `hard`, not the command.